compiler/src/main/java/com/squareup/anvil/compiler/codegen/PsiUtils.kt

```diff
@@ -411,3 +415,18 @@ fun KtUserType.findExtendsBound(): List<FqName> {
     .typeParameters
     .mapNotNull { it.fqName }
 }
+
+fun KtClassOrObject.singleInjectConstructorOrNull(fqName: FqName): KtConstructor<*>? {
```

Comment:
The naming is a bit odd, since Kotlin stdlib functions that end with `OrNull` aren't expected to throw an exception...perhaps some kdoc that clarifies the "returns null when 0 matching constructors found, throws when more than 1 found" behaviour would be enough